guidemaker-ember-template/src/components/search-input.hbs

```diff
@@ -1,61 +1,67 @@
-{{! template-lint-disable no-action }}
-{{#if this.searchService.index}}
-  <input
-    id="search-input"
-    type="search"
-    value={{this.value}}
-    oninput={{perform this.search value="target.value"}}
-    placeholder="Search the guides"
-    autocomplete="off"
-    onfocus={{action "onfocus"}}
-    onblur={{action "onblur"}}
-    data-test-search-input
-  />
+<div class="search-input" ...attributes>
+  {{#if this.searchService.index}}
+    <input
+      id="search-input"
+      type="search"
+      value={{this.value}}
+      oninput={{perform this.search value="target.value"}}
+      placeholder="Search the guides"
+      autocomplete="off"
+      data-test-search-input
+      {{on "focus" this.onfocus}}
+      {{on "blur" (perform this.closeMenu)}}
+    />
 
-  {{! Search results dropdown }}
-  <EmberTether
-    @target="#search-input"
-    @targetAttachment="bottom right"
-    @attachment="top right"
-    @constraints={{this._resultTetherConstraints}}
-    @class="ds-dropdown-results"
-  >
-    {{#if this.showDropdown}}
-      <div class="ds-suggestions ds-dropdown-menu">
-        <DropdownHeader>
-          Search Results
-        </DropdownHeader>
-
-        {{#each this.searchService.results as |result|}}
-          <SearchResult @result={{result}} />
-        {{else}}
-          <div class="algolia-docsearch-suggestion">
-            <div class="algolia-docsearch-suggestion--noresults">
-              <p>
-                No results found.
-                {{#if this.deprecationsGuideURL}}
-                  Try searching the
-                  <a href={{this.deprecationsGuideURL}} target="_deprecations">
-                    deprecations guide
-                  </a>.
-                {{/if}}
-              </p>
+    {{! Search results dropdown }}
+    <EmberTether
+      @target="#search-input"
+      @targetAttachment="bottom right"
+      @attachment="top right"
+      @constraints={{this._resultTetherConstraints}}
+      @class="ds-dropdown-results"
+    >
+      {{#if this.showDropdown}}
+        <div class="ds-suggestions ds-dropdown-menu" data-test-search-dropdown>
+          <DropdownHeader>
+            Search Results
+          </DropdownHeader>
+          {{#each this.searchService.results as |result|}}
+            <SearchResult @result={{result}} />
+          {{else}}
+            <div class="algolia-docsearch-suggestion">
+              <div
+                class="algolia-docsearch-suggestion--noresults"
+                data-test-search-noresults
+              >
+                <p>
+                  No results found.
+                  {{#if this.deprecationsGuideURL}}
+                    Try searching the
+                    <a
+                      href={{this.deprecationsGuideURL}}
+                      target="_deprecations"
```

Comment:
@mansona is `target="_deprecations"` a special guidemaker target?

Reply:
no this isn't something I'm aware of 😂 We can probably remove it 🤷